src/docs/design-review-guidelines.md

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Design review guidelines'
+---
+Please make sure to follow the following guidelines when applicable.
+
+There are multiple drivers for the formalization of V8's design reviews:
+
+1. Make it clear to Individual Contributors (ICs) who the decision makers are and highlight what the path forward is
+1. Create a forum to have straight-forward design discussions
+1. Reduce surprises on the Tech Lead (TL) layer
+1. Increase the involvement of all V8 contributors over the globe
+
+## Summary
+
+<figure>
+  <img src="/_img/docs/design-review-guidelines/design_review_guidelines.png" alt="">
```

Comment:
Can haz in SVG? Also please run `npm run img` afterwards.

Reply:
nit: we usually use dashes instead of underscores, i.e. `design-review-guidelines.svg`